src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -893,7 +893,6 @@ NuGet.Commands.TrustedSignersCommandRunner
 NuGet.Commands.TrustedSignersCommandRunner.ExecuteCommandAsync(NuGet.Commands.TrustedSignersArgs trustedSignersArgs) -> System.Threading.Tasks.Task<int>
 NuGet.Commands.TrustedSignersCommandRunner.TrustedSignersCommandRunner(NuGet.Packaging.Signing.ITrustedSignersProvider trustedSignersProvider, NuGet.Configuration.IPackageSourceProvider packageSourceProvider) -> void
 NuGet.Commands.UnexpectedDependencyMessages
-NuGet.Commands.UnresolvedMessages
```

Comment:
This class is made internal as per https://github.com/NuGet/NuGet.Client/pull/4149#discussion_r668345880 conversation